introduction.md

```diff
@@ -1 +1,3 @@
-Hi, my name is Jack. My favorite CodeSchool path so far is Git. No, wait... I meant to say JavaScript.
+* guitar fu
+* coding fu
+* and i can fly
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?